app/Http/Controllers/CourseController.php

```diff
@@ -40,4 +40,12 @@ public function index(int $courseId): Responsable
 
         return new SuccessResponse($courseOwnerIsChildAccountOfUdir);
     }
+
+    public function getStudentCount(string $courseId) {
```

Comment:
Method Naming: The name of the method getStudentCount() is not very descriptive. It might be better to name it something like getTotalStudentsCount() to make it clearer what the method does.